### PR TITLE
python3Packages.deploykit: init at 1.0.1

### PR DESCRIPTION
**Pull request by zowoq**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

cc @Mic92

### CODE REVIEW

**Comment by zowoq** [comment]
@ofborg build python3Packages.deploykit

File: pkgs/development/python-modules/deploykit/default.nix [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    repo = pname;
+    rev = version;
+    hash = "sha256-mQgv373BIlyNbm2+F6BqqdJkkJn4Cy5FNwQNAyBb/h4=";
+  };
+
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/numtide/deploykit/commit/44e3f12f2e65d11208e8809127dfe9a171a96db0.patch";
+      hash = "sha256-5Q9oRKhvVe3IrErHj/OTP3hKNsZCdRR0EV4XjdqA/us=";
+    })
```
Comment (Atemu) [inline pkgs/development/python-modules/deploykit/default.nix:27]: What's this patch for and should it be removed again at some point?

A comment would be helpful.

**Comment by Atemu** [comment]
Result of `nixpkgs-review pr 199257` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python310Packages.deploykit</li>
    <li>python39Packages.deploykit</li>
  </ul>
</details>

File: pkgs/development/python-modules/deploykit/default.nix (hunk 1 above)
Comment (Mic92) [reply pkgs/development/python-modules/deploykit/default.nix:27]: I just made another release 1.0.1... This means the patch can be dropped: https://github.com/numtide/deploykit/releases/tag/1.0.1

**Pull request merged**